README.md

```diff
@@ -1,2 +1,10 @@
 # About
-App for performing synthetic test CRUD operations
\ No newline at end of file
+App for performing synthetic test CRUD operations
+
+# Sample Git Text for Co-Authoring a Commit in CLI
```

Comment:
Have you tried git mob?

Reply:
I have not yet, it's a little ways down my to-do list but I could try to squeeze it in ;)